var/spack/repos/builtin/packages/py-particle/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyParticle(PythonPackage):
+    """Particle provides a pythonic interface to the Particle Data Group (PDG)
+    particle data tables and particle identification codes, with extended
+    particle information and extra goodies."""
+
+    git = "https://github.com/scikit-hep/particle.git"
+    url = "https://pypi.io/packages/source/p/particle/particle-0.11.0.tar.gz"
+    homepage = "https://github.com/scikit-hep/particle"
+
+    maintainers = ['vvolkl']
+
+    version('master', branch='master')
+    version('0.11.0', sha256='e90dc36c8b7d7431bd14ee5a28486d28b6c0708555845d1d7bdf59a165405f12')
+
+    depends_on('py-setuptools', type='build')
+
+    depends_on('py-attrs')
+    depends_on('py-hepunits')
+    depends_on('py-importlib-resources')
```

Comment:
These should be type=build/run and list required versions. There are also some missing dependencies depending on Python version:
```
python_requires = >=2.7, !=3.0, !=3.1, !=3.2, !=3.3, !=3.4
install_requires =
        enum34>=1.1; python_version<"3.4"
        typing>=3.7; python_version<"3.5"
        importlib_resources>=1.0; python_version<"3.7"
        attrs>=19.2
        hepunits>=1.2.0
```
For example, this would look like:
```python
depends_on('py-enum34@1.1:', when='^python@:3.3', type=('build', 'run'))
```

Reply:
Thanks for the comments. True, I didn't go into much detail with the dependencies, should be fixed now.